styles/index.css

```diff
@@ -0,0 +1,148 @@
+#top {
+    text-align : center;
+}
+body {
+    background-image: linear-gradient(to right, blue, rgb(0, 128, 255), white) ;
+    font-family:Georgia, 'Times New Roman', Times, serif;
+}
+
+input{
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+select {
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+
+button {
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+.boxes {
+    display: grid;
+    grid-template-columns: repeat(10,10%);
+    grid-template-rows: repeat(5,20%);
+}
+
+.flex_boxes_container {
+    background-color: antiquewhite;
+    grid-column : 2 /span 3;
+    grid-row: 1/ -1;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    
+}
+
+.flex_boxes {
+    border: 10px  rgb(0, 128, 255) solid;
+    padding: 10px ;
+    text-align: center;
+}
+
+#tempurature_box {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+}
+
+#weather_garden_box {
+    grid-column : 7 /span 3;
+    grid-row: 2/ span 3;
+    border: 1px solid #000;
+    text-align: center;
+    display: grid;
+    grid-template-columns: repeat(4,25%);
+    grid-template-rows: repeat(4,25%);
+    background-color: antiquewhite;
+}
+
+
+#weather_visuals {
+    /* background-color: rgb(131, 255, 127); */
+    grid-row: 2/ -1;
+    grid-column: 1/ -1;
+    display: grid;
+    grid-template-columns: repeat(4,25%);
+    grid-template-rows: repeat(4,25%);
+}
+
+#title{
+    grid-column: 1/-1;
+    grid-row: 1/2;
+    text-align: center;
+}
+
+#skyEmojis {
+    grid-row: 1/2;
+    grid-column: 1/-1;
+    font-size: xx-large;
+}
+
+#landscape {
+    grid-row: 4/-1;
+    grid-column: 1/-1;
+    font-size: x-large;
+}
+
+.cold {
+    color : teal;
+}
+
+.hot {
+    color : red;
+}
+
+.warm {
+    color : orange;
+}
+
+.cozy {
+    color : rgb(215, 206, 32);
+}
+
+.cool {
+    color : green;
+}
+
+.sun {
+    background-image: url(https://www.vmcdn.ca/f/files/sudbury/uploadedImages/news/localNews/2019/07/070719_kf_sun_sized.jpg;w=960);
+    background-position: center;
+    background-size: 100%;
+}
```

Comment:
Very cool using the images! In the future, it's a bit better practice to have the image stored on your own server instead of linking to someone else's. This way it won't disappear if the other server takes the image down. It also means that you're not putting load on someone else's server each time your website is viewed. The practice of using someone else's hosted image is called "hotlinking" and it's generally considered poor etiquette. No big harm here, but something to consider for the future 🙂